torch/_meta_registrations.py

```diff
@@ -2373,9 +2373,10 @@ def _formula_transposed(ln: int, p: int, d: int, k: int, s: int, op: int) -> int
             ret_shape.append(
                 _formula(dims[i], padding[i], dilation[i], kernel_size[i], stride[i])
             )
+    from torch.fx.experimental.symbolic_shapes import sym_or
 
     torch._check(
-        any(x > 0 for x in ret_shape[2:]),
+        sym_or(*[x > 0 for x in ret_shape[2:]]),
```

Comment:
nit:
```suggestion
        sym_or(*(x > 0 for x in ret_shape[2:])),
```
The original way (with square brackets) creates an intermediate list which is then converted into a tuple for the varargs. This way just creates the tuple directly.

Reply:
How does this avoid the DDE? Does `torch._check` not evaluate syms as bool or something weird? Or is it that you won't get a DDE as long as ONE of the ret_shapes are definitely > 0 so the others don't get evaluated?